phpunit.xml.dist

```diff
@@ -18,13 +18,17 @@
     </testsuites>
 
     <!-- Setup a listener for fixtures -->
-    <listeners>
+    <!-- <listeners>
```

Comment:
It would be good to remove this if not used

Reply:
Removed